src/mount/mount.ceph.c

```diff
@@ -201,8 +201,6 @@ static char *parse_options(const char *data, int *filesys_flags)
 			skip = 0;
 		} else {
 			skip = 0;
-			if (verboseflag)
-				printf("ceph: Unknown mount option %s\n",data);
```

Comment:
Now that I look, I don't know... the verboseflag was set, so I think it's entirely reasonable to inform that the mount helper did not recognize the option. If we're worried about those sorts of messages then we shouldn't specify -v when mounting.

The message could be more clear though. For one thing, we should probably make it clear that it comes from "mount.ceph" and not "ceph". It should also go to stderr, IMO. Maybe:

`fprintf(stderr, "mount.ceph: unrecognized mount option \"%s\", passing to kernel.\n", data);`

...just to make it clear that it's not necessarily a problem.